src/main/java/carsharing/CarSharingEngine.java

```diff
@@ -0,0 +1,90 @@
+package carsharing;
+
+import carsharing.customer.CustomerDAO;
+import carsharing.customer.CustomerDAOImpl;
+import carsharing.roles.CustomerRole;
+import carsharing.roles.ManagerRole;
+import carsharing.configuration.DBManager;
+
+import java.util.Scanner;
+
+public class CarSharingEngine {
+
+    public static final String BACK = "0. Back";
+    private final Scanner scanner;
+    private final CustomerDAO customerDao;
+    private String pathToDb;
+    private final DBManager DBManager;
```

Comment:
why do we need it?

Reply:
It moved this to constructor. 
I need it to pass the dbName to DBManager